src/main/plugins/org.dita.pdf2/xsl/fo/commons.xsl

```diff
@@ -459,6 +459,9 @@ See the accompanying license.txt file for applicable licenses.
 
     <xsl:template name="processTopicNotices">
         <fo:page-sequence master-reference="body-sequence" xsl:use-attribute-sets="page-sequence.notice">
+            <xsl:if test="key('map-id', ancestor-or-self::*[contains(@class, ' topic/topic ')][1]/@id)/ancestor::*[contains(@class,' bookmap/backmatter ')]">
```

Comment:
Another option would be to duplicate the `fo:page-sequence` element, using different attribute sets for back matter vs front matter. That would probably be the cleanest way to handle this while 1) keeping all values in attribute sets, 2) not affecting existing configurations, and 3) fixing the bug.